rca/api_content/content.py

```diff
@@ -100,8 +100,11 @@ def parse_items_to_list(data, type):
 
         if "social_image" in data and data["social_image"]:
             social_image = data["social_image"]["meta"]["detail_url"]
-            social_image = requests.get(url=social_image, timeout=10)
-            social_image = social_image.json()
+            social_image = fetch_data(url=social_image)
+            try:
+                social_image = social_image.json()
```

Comment:
Would it be worth calling `.json()` in the  `fetch_data()` function before its returned? This would keep calling code ignorant of a data structure it probably doesn't need to know about?